realsense2_camera/test/post_processing_filters/test_align_depth.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import sys
+import subprocess
+import time
+
+import pytest
+from launch import LaunchDescription
+import launch_ros.actions
+import launch_pytest
+import rclpy
+from rclpy import qos
+from rclpy.node import Node
+from sensor_msgs.msg import Image as msg_Image
+from sensor_msgs.msg import Imu as msg_Imu
+
+sys.path.append(os.path.abspath(os.path.dirname(__file__)+"/../utils"))
+import pytest_rs_utils
+from pytest_rs_utils import launch_descr_with_yaml
+from pytest_rs_utils import get_rosbag_file_path
+
+
+'''
+This test imitates the ros2 launch rs_launch.py realsense2_camera with the given parameters below
+Full command to reproduce locally
+ros2 launch rs_launch.py realsense2_camera camera_name:=camera enable_color:=true \
+    enable_depth:=true rgb_camera.profile:=1280x720x30 depth_module.profile:=640x480x30 \
+    align_depth.enable:true rosbag_filename:=`realpath outdoors_1color.bag`
+
+Then we check if these topics exist:
+/camera/color/image_raw
+/camera/depth/image_rect_raw
+/camera/aligned_depth_to_color/image_raw
+
+Also we check that the recieved frames of each topic are in the right width and height
+'''
+test_params = {
+    "rosbag_filename":get_rosbag_file_path("outdoors_1color.bag"),
+    'camera_name': 'camera',
```

Comment:
Good to use a unique name for camera. If there are many tests in one py file, the test may run in parallel and it may influence each other